CareKitStore/CareKitStore/CoreData/OCKStore+CarePlans.swift

```diff
@@ -134,6 +130,60 @@ extension OCKStore {
         }
     }
 
+    // MARK: Internal
+    // These methods are called from elsewhere in CareKit, but must always be called
+    // from the `contexts`'s thread.
+
+    func createCarePlansWithoutCommitting(_ plans: [Plan]) throws -> [Plan] {
+        try self.validateNew(OCKCDCarePlan.self, plans)
+        let persistablePlans = plans.map(self.createCarePlan)
+        let addedPlans = persistablePlans.map(self.makePlan)
+        return addedPlans
+    }
+
+    /// Updates existing tasks to the versions passed in.
+    ///
+    /// The copyUUIDs argument should be true when ingesting tasks from a remote to ensure
+    /// the UUIDs match on all devices, and false when creating a new version of a task locally
+    /// to ensure that the new version has a different UUID than its parent version.
+    ///
+    /// - Parameters:
+    ///   - tasks: The new versions of the tasks.
+    ///   - copyUUIDs: If true, the UUIDs of the tasks will be copied to the new versions
+    func updateCarePlansWithoutCommitting(_ plans: [Plan], copyUUIDs: Bool) throws -> [Plan] {
+        try validateUpdateIdentifiers(plans.map { $0.id })
+        try confirmUpdateWillNotCauseDataLoss(plans: plans)
```

Comment:
I think we can remove this for now, since the implementation doesn't actually do anything right now.

Reply:
Yup, I guess I was doing that as you were writing the comments 🙃

Reply:
Wow, you're just that good!